templates/ep19/bs/cfp/update_speakers.html

```diff
@@ -0,0 +1,34 @@
+{% extends "ep19/bs/base.html" %}
+
+{% load crispy_forms_tags static %}
+
+{% block morecss %}
+<link href="{% static "taggit_labels/css/taggit_labels.css" %}" type="text/css" media="all" rel="stylesheet" />
+{% endblock %}
+
+{% block morejs %}
+<script src="{% static "/js/taggit_labels_customised_with_max_number_of_tags.js" %}"></script>
```

Comment:
This is not needed for speakers, same for the CSS above.

Reply:
Thanks for catching this, removing!